http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/StatusService.scala

```diff
@@ -38,15 +33,14 @@ class StatusService(
   import dbRef._
 
   private val healthMonitor =
-    system.actorOf(HealthMonitor.props(Set(GoogleDataproc, Sam, Database))(() => checkStatus()))
+    system.actorOf(HealthMonitor.props(Set(Sam, Database))(() => checkStatus()))
   system.scheduler.scheduleWithFixedDelay(initialDelay, pollInterval, healthMonitor, HealthMonitor.CheckAll)
 
   def getStatus(): Future[StatusCheckResponse] =
     (healthMonitor ? GetCurrentStatus).asInstanceOf[Future[StatusCheckResponse]]
 
   private def checkStatus(): Map[Subsystem, Future[SubsystemStatus]] =
     Map(
-      GoogleDataproc -> checkGoogleDataproc(),
```

Comment:
Yup - thinking it doesn't really make sense to check all Google subsystems (since today we use much more than just Dataproc). So this service can only check subsystems that we operate (e.g. Database, Sam).